### PR TITLE
Fixed input error blocking commit issue when creating Deployment port

### PR DESCRIPTION
**Pull request by zjxzjw**
issue: https://github.com/rancher/dashboard/issues/18498

### Summary
Fixes #
-Change the valid range of Node Port from 1-65535 to 30000-32767
-Update the verification message in the English translation file
-Update the verification message in the Chinese translation file
-Fix the save logic when creating resources, add ID existence check
-Add test cases to cover validation scenarios where the Node Port is out of range
-Update Node Port example values in existing test cases

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [ ] The PR has a Milestone 
- [ ] The PR template has been filled out
- [ ] The PR has been self reviewed 
- [ ] The PR has a reviewer assigned
- [ ] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [ ] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes
- [ ] The PR has been reviewed in terms of Accessibility
- [ ] The PR has considered, and if applicable tested with, the three Global Roles `Admin`, `Standard User` and `User Base`

### CODE REVIEW

File: shell/mixins/create-edit-view/impl.js [hunk 1]
```diff
@@ -193,7 +193,7 @@ export default {
     },
 
     async actuallySave(url) {
-      if ( this.isCreate ) {
+      if ( this.isCreate && !this.value?.id ) {
```
Comment (richard-cox) [inline shell/mixins/create-edit-view/impl.js]: Little bit surprised at this one. Is it needed?

If so it feels like there's a bug somewhere higher up the chain

Comment (zjxzjw) [reply shell/mixins/create-edit-view/impl.js]: Thank you for the review. Sorry, I did not consider that it would cause upstream bugs. I have fixed this issue again and please review it again. The purpose is to solve the problem of inputting the wrong port, correcting the port, and clicking submit again to automatically select the request method

Comment (richard-cox) [reply shell/mixins/create-edit-view/impl.js]: I think the issue is understanding what bug the change resolves as it doesn't seem related to the issue. If there is a bug in this part think it would need solving where isCreate is determined rather than here

Comment (zjxzjw) [reply shell/mixins/create-edit-view/impl.js]: I understand your question now. This change is to fix the configuration of service when creating a deployment I came to expose the container, but the listening port is limited to 3000-32767. However, during my use, I found a problem that violates human operating habits. I entered the wrong port and clicked "Create". The resource has already been created, but the page prompts me that the port I entered is incorrect. At this time, I am still on the "Create" page, and the button displays "Create" instead of "Save". When I modify the correct port, clicking the "Create" button again prompts me that the request method is incorrect. The normal process should be to click the "Modify" button again, and the corresponding text should display "Save" instead of "Request Method Error". However, I have not found a solution to the problem yet. I hope you can assist me in handling this PR properly
